licenses.d2iq.yaml

```diff
@@ -385,7 +385,7 @@ resources:
       - license_path: LICENSE
         ref: ${image_tag}
         url: https://github.com/stakater/Reloader
-  - container_image: ghcr.io/mesosphere/dkp-container-images/nvcr.io/nvidia/cloud-native/gpu-operator-validator:v24.3.0-d2iq.0
+  - container_image: ghcr.io/mesosphere/dkp-container-images/nvcr.io/nvidia/cloud-native/gpu-operator-validator:v24.6.1
```

Comment:
This image doesn't exist, revert back to use of non-patched version: `nvcr.io/nvidia/cloud-native/gpu-operator-validator:v24.6.1`